docs/DeveloperGuide.md

```diff
@@ -769,9 +769,42 @@ testers are expected to do more *exploratory* testing.
    3. Test case: `deletestudent 0`<br>
       Expected: No student is deleted. Error details shown in the status message.
 
-   4. Other incorrect delete commands to try: `deletestudent`, `deletestudent x`, `...` (where x is larger than the list size)<br>
+   4. Other incorrect delete student commands to try: `deletestudent`, `deletestudent x`, `...` (where x is larger than the list size) <br>
       Expected: Similar to previous.
 
+### Editing a student
+
+1. Edit a student to change name and/or matric number
+
+    1. Prerequisites: List all students using the `liststudent` or `listall` command. There must be at least one student in the list. 
+
+    2. Test case: `editstudent 1 s/John` <br>
+       Expected: First student's name is changed to John for all instances. Matric number remains the same. Details of the new student name is shown in the status message.
+
+    3. Test case: `editstudent 0 s/John` <br>
+       Expected: No student's details is changed. Error details shown in the status message.
+
+    4. Other incorrect edit student commands to try: `editstudent x s/John`, editstudent 1 s/John m/B1234567A`, `...` (where x is larger than the list size) <br>
+       Expected: Similar to previous.
+ 
+### Finding a student
+
+1. Find a student based on their names
+
+    1. Prerequisites: List all students using the `liststudent` or `listall` command. There must be at least one student in the list.
+   
+    2. Test case: `findstudent yeoh` <br>
+       Expected: Students' whose name contains `yeoh` regardless of capitalisation will be shown in the student list. Number of students listed is shown in the status message.
+   
+    3. Test case: `findstudent` <br>
+       Expected: No student is found. Error details shown in the status message.
+   
+    4. Test case: `findstudent x` (where x does not exist in any students' name in student list) <br> 
+       Expected: No student is found. Student list will be empty. Zero students listed is shown in the status message.
```

Comment:
I think can just stick to `Error details shown in the status message.` for consistency

Reply:
For this, it isnt an error, is really no students found and blank list

Reply:
That makes sense